planet.rb

```diff
@@ -0,0 +1,59 @@
+class Planet
+  def initialize(name, color, mass_kg, distance_from_sun_km, fun_fact)
+    @name = name
+    @color = color
+    @mass_kg = mass_kg
+    @distance_from_sun_km = distance_from_sun_km
+    @fun_fact = fun_fact
+  end
+
+  def name
+    return @name
+  end
```

Comment:
You should use the helper methods (`attr_reader`, `attr_writer`, `attr_accessor`) to generate these getter and setter methods.